src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/SchedulesService.kt

```diff
@@ -48,6 +48,11 @@ class SchedulesService(
       throw e
     }
   }
+
+  suspend fun deleteUnknownMappings() =
+    activitiesNomisApiService.getMaxCourseScheduleId().also { maxCourseScheduleId ->
+      mappingService.deleteMappingsGreaterThan(maxCourseScheduleId)
+    }
```

Comment:
This does the actual work - get the highest `CRS_SCH_ID` from NOMIS preprod and delete any mappings with a greater value.